### PR TITLE
Fix for video settings form.

### PR DESCRIPTION
**Pull request by michelinewu**

### CODE REVIEW

File: app/services/settings-v2/video.ts [hunk 1]
```diff
@@ -47,60 +70,105 @@ export class VideoSettingsService extends StatefulService<IVideoSetting> {
...
+    };
+  }
+
+  migrateSettings(display?: TDisplayType) {
+    this.SET_VIDEO_CONTEXT(display, this.contexts[display].video);
+
+    // if this is the first time starting the app
+    // set default settings for horizontal context
+    if (!this.greenService.views.videoSettings.horizontal) {
+      console.log('setting horizontal');
```
Comment (gettinToasty) [inline app/services/settings-v2/video.ts]: console log

**Review by gettinToasty — CHANGES_REQUESTED** [review]

File: app/services/scene-collections/nodes/root.ts [hunk 2]
```diff
@@ -55,7 +55,7 @@ export class RootNode extends Node<ISchema, {}> {
       transitions,
       hotkeys,
       guestCam,
-      baseResolution: this.videoSettingsService.baseResolution,
+      baseResolution: this.videoService.baseResolution,
```
[the remark this answers is not in the document]
Comment (michelinewu) [reply app/services/scene-collections/nodes/root.ts:58]: I'm now passing the base resolution through the video service in the next migration so I figured I'd just do it here too.

File: app/services/settings-v2/video.ts (hunk 1 above)
Comment (michelinewu) [reply app/services/settings-v2/video.ts]: Removed

File: app/services/settings-v2/video.ts [hunk 3]
```diff
@@ -150,36 +215,57 @@ export class VideoSettingsService extends StatefulService<IVideoSetting> {
...
+      }
+      case 'green': {
+        this.contexts.green.video = this.state.green;
+        break;
+      }
+      default: {
+        this.contexts.horizontal.video = this.state.horizontal;
+      }
+    }
+  }
```
[the remark this answers is not in the document]
Comment (michelinewu) [reply app/services/settings-v2/video.ts:231]: Updated

[the remark this answers is not in the document]
Comment (gettinToasty) [reply app/services/settings-v2/video.ts:219]: still needs to update legacySettings though right?

**Review by gettinToasty — APPROVED** [review]

**Pull request merged**